src/utils.cr

```diff
@@ -29,7 +28,6 @@ macro relative_{{method.id}}(route, &block)
     namespace = \{{block.args[-1]}}.params.query["namespace"]? || DPPM::Prefix.default_group
     \{{block.body}}
   end
-  \{% debug %}
   puts "built route at {{method.upcase.id}}:\t#{fmt_route(\{{route}})}"
```

Comment:
this line can go too -- they were there to track down #12, and I just hadn't gotten to cleaning them up yet. Thanks for taking care of that for me